doitlive.py

```diff
@@ -19,12 +18,18 @@
 import getpass
 import socket
 import subprocess
+import click
 from code import InteractiveConsole
 from tempfile import NamedTemporaryFile
 from collections import OrderedDict
-
-import click
-from click import echo, style, secho, getchar
+from termutils import raw_mode
```

Comment:
I agree regarding the grouping, on relative vs absolute I thought this would apply:
>Relative imports for intra-package imports are highly discouraged. Always use the absolute package path for all imports.